src/Init/System/FilePath.lean

```diff
@@ -19,16 +19,12 @@ def pathSeparator : Char :=
 def pathSeparators : List Char :=
   if isWindows then ['\\', '/'] else ['/']
 
-/-- The character that is used to separate the entries in the $PATH environment variable. -/
+/-- The character that is used to separate the entries in the $PATH (or %PATH%) environment variable. -/
 def searchPathSeparator : Char :=
   if isWindows then ';' else ':'
 
-/-- The list of all possible separators. -/
-def searchPathSeparators : List Char :=
-  if isWindows then [';', ':'] else [':']
```

Comment:
I don't think `:` should be a path separator on Windows. Consider that `%PATH%` usually looks like `C:/foo/bin;C:/bar/bin` etc. but the colons shouldn't be split.

Reply:
Makes sense to me.